options/posix/generic/lookup.cpp

```diff
@@ -86,10 +88,27 @@ int lookup_name_dns(struct lookup_result &buf, const char *name,
 	// for dns
 	sin.sin_port = htons(53);
 
-	auto nameserver = get_nameserver();
-	if (!inet_aton(nameserver ? nameserver->name.data() : "127.0.0.1", &sin.sin_addr)) {
-		mlibc::infoLogger() << "lookup_name_dns(): inet_aton() failed!" << frg::endlog;
-		return -EAI_SYSTEM;
+	int nameserver_idx = 0;
+	while (true) {
+		auto nameserver = get_nameserver(nameserver_idx++);
+		if (!nameserver) {
+			// no more nameservers in resolf.conf,
```

Comment:
Another style note: comments should be formally written. So with proper punctuation and capitalization.